docs/installation/windows.md

```diff
@@ -9,122 +9,296 @@ parent = "smn_engine"
 <![end-metadata]-->
 
 # Windows
-> **Note:**
-> Docker has been tested on Windows 7 and 8.1; it may also run on older versions.
-> Your processor needs to support hardware virtualization.
 
-The Docker Engine uses Linux-specific kernel features, so to run it on Windows
-we need to use a lightweight virtual machine (VM).  You use the **Windows Docker
-Client** to control the virtualized Docker Engine to build, run, and manage
-Docker containers.
+You install Docker using Docker Toolbox. Docker Toolbox includes the following Docker tools:
 
-To make this process easier, we've designed a helper application called
-[Boot2Docker](https://github.com/boot2docker/boot2docker) which creates a Linux virtual
-machine on Windows to run Docker on a Linux operating system.
+* Docker Machine binary for running the `docker-machine` binary
+* Docker Engine binary for running the `docker` binary
```

Comment:
binary twice; you can copy this from the Mac docs

(Didn't review the Windows docs the previous time)